mixins.scss

```diff
@@ -875,7 +930,7 @@
 	@each $value in $values {
 
 		// If the value is zero or of a type that doesn’t need conversion, return the value untouched
-		@if ( $value == 0 ) or type_of($value) == string or type_of($value) == color or type_of($value) == bool {
+		@if index(auto inherit initial none 0, $value) or type-of($value) != "number" {
```

Comment:
I would change that in another PR, because i will concentrate here only on the documentation